### PR TITLE
Add marquee and another gif

### PR DESCRIPTION
**Pull request by javorszky**

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -2,18 +2,26 @@
...
+         width="158"
+         alt="Still under construction">
     <div>
       <img src="/constjackhammer_e0.gif" alt="An animated construction worker operating a jackhammer.">
       <img src="/anim0206-1_e0.gif" alt="A cartoon construction worker eating alternating between eating a sandwich and drinking water.">
       <img src="/constjackhammer_e0.gif" alt="An animated construction worker operating a jackhammer.">
       <img src="/anim0206-1_e0.gif" alt="A cartoon construction worker eating alternating between eating a sandwich and drinking water.">
       <img src="/constjackhammer_e0.gif" alt="An animated construction worker operating a jackhammer.">
     </div>
-
```
Comment (raaims) [inline pages/index.vue]: please bring back the whitespace

Comment (javorszky) [reply pages/index.vue]: fixed in https://github.com/fimion/icannotdesign/pull/1/commits/c7ac4049b27de8ed47c7219ffc2d058c5da14b33

File: pages/index.vue [hunk 2]
```diff
@@ -2,10 +2,20 @@
   <div class="main">
     <h1 class="title">I Cannot Design.</h1>
     <hr>
+    <marquee direction="down" width="250" height="200" behavior="alternate">
+      <marquee behavior="alternate">
+        Under construction!!!1!1one
+      </marquee>
+    </marquee>
```
Comment (fimion) [inline pages/index.vue]: Make sure to give the text here a background color so it is accessible. Might I suggest magenta?

We're about poor design choices, not poor accessibility. I want everyone to be able to enjoy this.

**Review by fimion — CHANGES_REQUESTED** [review]
Looks terrible, i love it! one change requested.

File: pages/index.vue (hunk 2 above)
Comment (javorszky) [reply pages/index.vue]: Good call, let me add some additional things

**Review by fimion — APPROVED** [review]
Approved!
Approved

**Comment by javorszky** [comment]
There, now it's done

**Pull request merged**